app/src/main/java/io/pumpkinz/pumpkinreader/model/Item.java

```diff
@@ -26,37 +27,51 @@ public Item(int id, boolean deleted, String type, String by, long time, String t
         this.dead = dead;
     }
 
-    /** The item's unique ID */
+    /**
+     * The item's unique ID
```

Comment:
Please remove all these accessor / mutator comments.
They are simple and self explanatory.
Additional comments are most unecessary.